### ISSUE TITLE
WCAG 1.4.3: Elements must have sufficient color contrast (.active[aria-current="page"][href$="extensions"])

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2341]
#### Issue
`Elements must have sufficient color contrast` ([color-contrast](https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=msftAI))
#### Target application
[Extensions - Microsoft MakeCode](https://makecode.microbit.org/extensions)
#### Element path
.column > nav > .active[aria-current="page"][href$="extensions"]
#### Snippet
`<a class="active section" href="/extensions" aria-current="page">Extensions</a>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 3.98 (foreground color: #4183c4, background color: #ffffff, font size: 12.0pt, font weight: bold). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 75.0.3770.142
This accessibility issue was found using Accessibility Insights for Web 2.7.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

this is the docs breadcrumb:

<img width="196" alt="Screen Shot 2019-08-02 at 3 48 07 PM" src="https://user-images.githubusercontent.com/5615930/62402511-f388ba00-b53c-11e9-876b-84e3cb9c4c68.png">

**Comment by abchatra** [comment]
fixed
